src/mod_adhoc.erl

```diff
@@ -31,11 +31,15 @@
 -type command_hook_acc() :: {error, exml:element()} | exml:element() | ignore | empty.
 -export_type([command_hook_acc/0]).
 
+%% Gen_mod callbacks
```

Comment:
```suggestion
%% gen_mod callbacks
```

Reply:
This is the same comment as in `mod_last` so I decided we could merge it like it is. I could fix both with some upcoming PR.